src/components/tree/infoPanels/click.js

```diff
@@ -59,6 +60,33 @@ const Link = ({url, title, value}) => (
   </tr>
 );
 
+/**
+ * Render a 2-column table of gene -> mutations.
+ * Rows are sorted by gene name, alphabetically, with "nuc" last.
+ * Mutations are sorted by genomic position.
+ * todo: sort genes by position in genome
+ * todo: provide in-app links from mutations to color-bys? filters?
+ */
+const MutationTable = ({mutations}) => {
+  const geneSortFn = (a, b) => {
+    if (a[0]==="nuc") return 1;
+    if (b[0]==="nuc") return -1;
+    return a[0]<b[0] ? -1 : 1;
+  };
+  const mutSortFn = (a, b) => {
+    const [aa, bb] = [parseInt(a.slice(1, -1), 10), parseInt(b.slice(1, -1), 10)];
+    return aa<bb ? -1 : 1;
+  };
+  return (
+    Object.entries(mutations)
+      .sort(geneSortFn)
+      .map(([gene, muts], index) => (
+        item(index === 0 ? "Mutations from root" : "", gene + ": " + [...muts].sort(mutSortFn).join(", ").substring(0, 200))
```

Comment:
![Screen Shot 2021-01-29 at 4 57 44 PM](https://user-images.githubusercontent.com/40774762/106341750-32dbb800-6253-11eb-9b11-16c892f02a3e.png)
With this substring, the list of mutations gets randomly cut off when it's too long. 

If we want to limit the number mutations displayed, it might be better to slice the array first:
```javascript
const mutLimit = 30;
[...muts].sort(mutSortFn).slice(0, mutLimit).join(", ") + (muts.length > mutLimit && "...")
```

Reply:
That's definitely nicer.

Reply:
If we are cutting off the display of mutations, we should also make this explicit in the box - so people don't think it's a complete list erroneously. (Apologies if this is integrated but doesn't show up in Jover's example above!)

Reply:
@trvrb given that #1259 asks to "display [the] full list of AA changes" why would we truncate this list? I changed back to displaying all AA muts in 4fee777 as I though this would be the expected behaviour - am I missing something?

Reply:
Try this out in dengue. It ends up being pages long. Good to truncate in this case.

Reply:
Got it. This makes sense. Thanks James. I just spent some more time with this PR and I think that the scrolly overlay panel is a fine solution. Having "mutations from root" at the end of the list of fields is helpful here. I'd agree that (2) is preferable to (1) for the moment.